src/main/java/com/example/flightplanner/Controllers/TestController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.flightplanner.Controllers;
+
+import com.example.flightplanner.Repositories.FlightRepository;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/testing-api")
+public class TestController {
+
+    FlightRepository flightRepository;
+
+    public TestController(FlightRepository flightRepository) {
+        this.flightRepository = flightRepository;
+    }
+
+    @RequestMapping("/clear")
+    public void clearFlights(){
+        if(!flightRepository.flights.isEmpty()){
```

Comment:
Šitā gan nevajadzētu - piekļūt pa taisno `repository.flights`. Liec to flights iekš repository privātu.
Otra lieta - šo funkcionalitāti jāpārnes uz repository, tā ka controller tikai izsauc metodi no service (šobrīd te ir repository, bet izlasi nākamo punktu).
Trešā lieta - es tomēr neieteiktu lietot repository pa taisno iekš controller - labāk ieliec iekš service metodi clearFlights(), kura izsauks repository clearFlights()